From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: chao zeng <chao.zeng@siemens.com>
Date: Fri, 11 Nov 2022 11:17:47 +0800
Subject: [PATCH] net:ti:icssg: select NET_DEVLINK

NET_DEVLINK is necessary for current icssg driver.
However it has not be enabled.
This configuration should report to TI.

Signed-off-by: chao zeng <chao.zeng@siemens.com>
---
 drivers/net/ethernet/ti/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/ti/Kconfig b/drivers/net/ethernet/ti/Kconfig
index 90ede5a5f108..7830994c5a55 100644
--- a/drivers/net/ethernet/ti/Kconfig
+++ b/drivers/net/ethernet/ti/Kconfig
@@ -184,6 +184,7 @@ config TI_ICSSG_PRUETH
 	select TI_DAVINCI_MDIO
 	select NET_PTP_CLASSIFY
 	select TI_ICSS_IEP
+	select NET_DEVLINK
 	imply PTP_1588_CLOCK
 	depends on PRU_REMOTEPROC
 	depends on NET_SWITCHDEV
